### PR TITLE
adds an is_pro flag to the profile, so that we can easily toggle users back and forth into pro features

### PR DESCRIPTION
**Pull request by owocki**
##### Description

adds an is_pro flag to the profile, so that we can easily toggle users back and forth into pro features

##### Refers/Fixes

https://gitcoincore.slack.com/archives/CCL5Q0G5S/p1611589993005000

##### Testing

tested locally

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

File: app/dashboard/models.py [hunk 1]
```diff
@@ -397,6 +397,7 @@ class Bounty(SuperModel):
         default=False, help_text=_('This bounty will be part of the hypercharged bounties')
     )
     hyper_next_publication = models.DateTimeField(null=True, blank=True)
+    is_pro = models.BooleanField(help_text=_('Is this use upgraded to pro?'))
```
Comment (thelostone-mc) [inline app/dashboard/models.py:400]: *user

Comment (octavioamu) [reply app/dashboard/models.py:400]: I think this need to be on profile not on the bounty model as context is looking for `profile.is_pro` Unless Im missing something.

**Review by octavioamu — CHANGES_REQUESTED** [review]

File: app/dashboard/models.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/models.py:400]: oh shoot ur right

Comment (owocki) [reply app/dashboard/models.py]: coding is hard, thats why im a powerpoint guy these days.

fixed at ba14183f3f0016013058cbfc136ec505df2b9023

**Comment by owocki** [comment]
@thelostone-mc @octavioamu fixed! ba14183f3f0016013058cbfc136ec505df2b9023

**Review by aamustapha — APPROVED** [review]

**Review by chibie — APPROVED** [review]

**Comment by thelostone-mc** [comment]
conflicts + missing default value + migration out of date!
pushing a commit to fix those and merging into master

**Pull request merged**